MapboxNavigation/NavigationView.swift

```diff
@@ -152,64 +135,41 @@ open class NavigationView: UIView {
     }
     
     func commonInit() {
+        accessibilityIdentifier = "NavigationViewRoot"
```

Comment:
Should we leave these in?

Reply:
This view is straight forward to find with XCUI w/o an identifier so let's drop it from this PR